static/scripts/files.js

```diff
@@ -57,6 +57,8 @@ $(document).ready(() => {
 		$moveModal.modal('hide');
 		if (textStatus === 'timeout') {
 			$.showNotification('Zeitüberschreitung der Anfrage', 'warn');
+		} else if (req.responseJSON.error.message === 'No file service available.') {
```

Comment:
work with error codes instead, check that `req.responseJSON.error` exists before trying to access `.message`